python/ray/_private/ray_logging/constants.py

```diff
@@ -41,6 +41,9 @@ class LogKey(str, Enum):
     NODE_ID = "node_id"
     ACTOR_ID = "actor_id"
     TASK_ID = "task_id"
+    ACTOR_NAME = "actor_name"
+    TASK_NAME = "task_name"
+    TASK_FUNCTION = "task_function"
```

Comment:
nit: task_function_name. If you think it's too long, we can also do task_func_name

Reply:
Synced offline. Changing to function name will implicitly indicate the parameter type and will be helpful to understand the field.